modules/ROOT/pages/index.adoc

```diff
@@ -51,7 +51,7 @@ If the app doesn't specify a value for `entryTtl`, the default value depends on
 +
 Object stores have a rolling TTL by default.
 +
-Accessing the data at least once a week extends the TTL for 30 days.
+Accessing the data during the last seven days of a 30-day window extends the TTL for another 30 days.
```

Comment:
It may be safer to not include "30-day window" : "Accessing the data during the last seven days of the item TTL extends the TTL for another 30 days".
@hannanelson what are your thoughts on this?

Reply:
This comment is no longer relevant after the offline discussions we had within the team. Resolving it.